lua/snippy.lua

```diff
@@ -144,23 +144,43 @@ end
 local function get_snippet_at_cursor()
     M.read_snippets()
     local _, col = unpack(api.nvim_win_get_cursor(0))
-    local current_line = api.nvim_get_current_line()
-    local word = current_line:sub(1, col):match('(%S+)$')
-    if word then
+
+    -- remove leading whitespace for current_line_to_col
```

Comment:
```suggestion
    -- Remove leading whitespace for current_line_to_col
```

I think I prefer the comments to start with a capital letter for consistency.